src/components/__tests__/MenuSelect.js

```diff
@@ -341,19 +338,46 @@ describe('custom default render', () => {
       },
       template: `
         <MenuSelect v-bind="props">
-          ${defaultSlot}
+          <template v-slot="{ items, canRefine, refine }">
+            <select
+              @change="refine($event.currentTarget.value)"
+              :disabled="!canRefine"
+            >
+              <option value="">All</option>
+              <option
+                v-for="item in items"
+                :key="item.value"
+                :value="item.value"
+                :selected="item.isRefined"
+                :data-test="item.value"
+              >
+                {{item.label}}
+              </option>
+            </select>
+          </template>
         </MenuSelect>
       `,
     });
 
-    const selected = wrapper.find('[value="Samsung"]');
-    const options = wrapper.findAll('option:not([value="Samsung"])');
+    // Unlike in vue 2, in vue 3 when value and textContent are the same,
+    // Vue doesn't render the value attribute at all.
+    // For example,
+
+    // In Vue 2,
+    // <option value="Apple">Apple</option>
+
+    // becomes, in Vue 3
+    // <option>Apple</option>
+
+    const selected = wrapper.find('[data-test="Samsung"]');
+    const options = wrapper.findAll('option:not([data-test="Samsung"])');
 
     expect(selected.element.selected).toBe(true);
 
-    options.wrappers.forEach(option => {
-      expect(option.element.selected).toBe(false);
-    });
+    for (let i = 0; i < options.length; i++) {
+      // `[i]` is for VTU@2, and `.at(i)` is for VTU@1.
```

Comment:
does the VTU2 version work for VTU1?

Reply:
It's exclusive.

`[i]` only for VTU@2 and `.at(i)` only for VTU@1.